src/lib/mcp.ts

```
@@ -0,0 +1,316 @@
+import { Client as MCPClient } from '@modelcontextprotocol/sdk/client/index';
+import { StdioClientTransport } from '@modelcontextprotocol/sdk/client/stdio';
+import { SSEClientTransport } from '@modelcontextprotocol/sdk/client/sse';
+import { WebSocketClientTransport } from '@modelcontextprotocol/sdk/client/websocket';
+import type { Transport as MCPTransport } from '@modelcontextprotocol/sdk/shared/transport';
+import type { Tool, CallToolRequest, CallToolResult, ListResourcesResult, ListPromptsResult } from '@modelcontextprotocol/sdk/types';
+import { z } from 'zod';
+import { EventEmitter } from 'events';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)